src/components/ProjectSettingsTabs/Context.tsx

```diff
@@ -7,6 +7,10 @@ import { ProjectSettingsTabRoutes } from "constants/routes";
 /* Given a diff between a form's previous and current state, check if it represents an array field added with no content. */
 const isArrayPushUpdate = (diff: object): boolean =>
   Object.values(diff).every((val) => {
+    if (val === null) {
+      return false;
+    }
+
     if (typeof val !== "object") {
       return false;
     }
```

Comment:
Maybe we can condense these two `if` statements into one since they both return `false`?  But I don't know if that would hurt readability 🤔 Your call!